roles/dellhw_exporter/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Install "dellhw_exporter"
+  run_once: true
+  kubernetes.core.k8s:
+    server_side_apply:
+      field_manager: "atmosphere.common"
+      force_conflicts: true
+    definition:
+      - apiVersion: v1
+        kind: Namespace
+        metadata:
+          name: "{{ dellhw_exporter_helm_release_namespace }}"
+
```

Comment:
Let's take advatange of the `createNamespace` feature inside the `install` section. and avoid this and make it managed by Flux.

Reply:
One question here, if namespace didn't exist, helmrepository cannot be created.